integration-tests/test-helper.js

```diff
@@ -258,6 +258,11 @@ export class TestRig {
 
             result = filteredLines.join('\n');
           }
+          // If we have stderr output, include that also
```

Comment:
I also added this so that stderr output will be included if present, even if the exit code is 0.